code/src/components/Header.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components/';
+import moment from 'moment';
+
+import { tasks } from 'reducers/tasks';
+
+const HeaderContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  padding: 20px;
+  background: #f4f4f4;
+`;
+
+const Title = styled.div`
+  font-size: 20px;
+  font-weight: bold;
+`;
+
+const Tasks = styled.div`
+  // font-size: 16px;
+  text-align: end;
+`;
+
+const Date = styled.div`
```

Comment:
looks like no special styling is used here. could this be replaced with a plain div tag?